Fix node placement test

nunnatsa opened the pull request:
Fix the node placement test. Remove the the VMI creation loop because
the VMIs are not related to KubeVirt workload configurations, and so,
it's not related to HCO workload configurations.

```jira-ticket
None
```

**Release note**:
```release-note
None
```

tiraboschi commented on tests/func-tests/monitoring_test.go:
```diff
@@ -153,6 +152,26 @@ func getHCO(client kubecli.KubevirtClient) v1beta1.HyperConverged {
+	var kv kubevirtcorev1.KubeVirt
+	ExpectWithOffset(
+		1,
+		client.RestClient().Get().
```
```kubevirt, err := client.KubeVirt(flags.KubeVirtInstallNamespace).Get("kubevirt-kubevirt-hyperconverged", &metav1.GetOptions{})``` should be enough

nunnatsa committed:
[Func test] Remove the VMI creation loop

Fix the node placement test. Remove the the VMI creation loop because
the VMIs are not related to KubeVirt workload configurations, and so,
it's not related to HCO workload configurations.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

tiraboschi commented:
/approve

nunnatsa commented:
/retest

tiraboschi commented:
/retest

tiraboschi commented:
/retest

tiraboschi commented:
/retest
